packaging/systemd/ncr.service

```diff
@@ -5,5 +5,9 @@ After=network.target
 [Service]
 ExecStart=/opt/ncr/ncr --config /etc/ncr/ncr.yml
 KillMode=process
-Restart=on-failure
+Restart=always
 User=ncr
+
+[Install]
+WantedBy=multi-user.target
+Alias=ncr.service
```

Comment:
this one was an early attempt to get the daemon to start on boot on systemd.
i'm really no systemd expert hehe.

the alias may not be needed. shall i remove it?